x-pack/plugins/security_solution/server/fleet_integration/fleet_integration.test.ts

```diff
@@ -429,12 +433,12 @@ describe('ingest_integration tests ', () => {
     });
   });
 
-  describe('package policy update callback (when the license is at least platinum)', () => {
+  describe('package policy update callback (when the license is at least enterprise)', () => {
```

Comment:
same thing here

Reply:
Test expecting platinum license is below. This was added were platinum test used to be.